### PR TITLE
Allow different hosts to have different interface prefixes.

### PR DESCRIPTION
**Pull request by plwhite**
This is allowed for mixed OpenStack / Docker interop.

### CODE REVIEW

**Comment by plwhite** [comment]
I'm testing a mixed OpenStack plus Docker demo - and it exposed some over-restrictive firewalling that I have loosened.

File: CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Reduce felix occupancy by replacing endpoint dictionaries by "struct"
   objects.
+- Allow different hosts to have different interface prefixes.
```
Comment (matthewdupre) [inline CHANGES.md]: I think it's worth specifying 'for combined OpenStack and Docker systems' or similar.

**Comment by plwhite** [comment]
Sorry - hold off on merge wait. UTs are clean, but manual test is showing problems.

**Comment by matthewdupre** [comment]
Sure - made one comment.  Ping me when it's ready.

**Pull request merged**